rspec-mode.el

```diff
@@ -225,6 +225,13 @@ info, are considered errors."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-holder-dirs '("app" "lib")
```

Comment:
> i don't care for applike. It is too rails centric.

Web application-centric, maybe? A Padrino app would have an `app` directory as well (and a Lotus app would have `apps`). FWIW, I don't like `applike` too much either, and I haven't done any Rails for more than 2 years, but I figured the concept would be familiar to most of the Ruby developers.

> Specs for files in the lib of a rails project would, optimally, go into spec/lib/.

Are you sure? That's the first time I'm hearing about this. The [RSpec docs](https://relishapp.com/rspec/rspec-rails/v/3-3/docs/directory-structure) (search for "Tips on Spec Location") suggest treating `app` and `lib` the same in this respect.

> Is the logic we want really something like: files below the "main" source dir get that dir name trimmed, all others have the same structure below the spec dir as they do below the project root?

As per above, no. But I want you to be able to have it, so a customization variable would be best. Otherwise, hardcoding the list `("app" "lib")` seems to be most appropriate.